internal/exec/help.go

```diff
@@ -20,43 +19,59 @@ func processHelp(
-				"before executing the command")
-			u.PrintMessage(" - 'atmos terraform apply' and 'atmos terraform deploy' commands support '--from-plan' flag. If the flag is specified, " +
-				"the commands will use the planfile previously generated by 'atmos terraform plan' command instead of generating a new planfile")
-			u.PrintMessage(" - 'atmos terraform apply' and 'atmos terraform deploy' commands commands support '--planfile' flag to specify the path " +
-				"to a planfile. The '--planfile' flag should be used instead of the planfile argument in the native 'terraform apply <planfile>' command")
-			u.PrintMessage(" - 'atmos terraform clean' command deletes the '.terraform' folder, '.terraform.lock.hcl' lock file, " +
-				"and the previously generated 'planfile', 'varfile', and 'backend.tf.json' file for the specified component and stack. " +
-				"Use the --everything flag to also delete the Terraform state files and directories for the component. " +
-				"Note: State files store the local state of your infrastructure and should be handled with care, if not using a remote backend.\n\n" +
-				"Additional flags:\n" +
-				"  --force Forcefully delete Terraform state files and directories without interaction\n" +
-				"  --skip-lock-file Skip deleting the '.terraform.lock.hcl' file\n\n" +
-				"If no component or stack is specified, the clean operation will apply globally to all components.")
-			u.PrintMessage(" - 'atmos terraform workspace' command first runs 'terraform init -reconfigure', then 'terraform workspace select', " +
-				"and if the workspace was not created before, it then runs 'terraform workspace new'")
-			u.PrintMessage(" - 'atmos terraform import' command searches for 'region' in the variables for the specified component and stack, " +
-				"and if it finds it, sets 'AWS_REGION=<region>' ENV var before executing the command")
-			u.PrintMessage(" - 'atmos terraform generate backend' command generates a backend config file for an 'atmos' component in a stack")
-			u.PrintMessage(" - 'atmos terraform generate backends' command generates backend config files for all 'atmos' components in all stacks")
-			u.PrintMessage(" - 'atmos terraform generate varfile' command generates a varfile for an 'atmos' component in a stack")
-			u.PrintMessage(" - 'atmos terraform generate varfiles' command generates varfiles for all 'atmos' components in all stacks")
-			u.PrintMessage(" - 'atmos terraform shell' command configures an environment for an 'atmos' component in a stack and starts a new shell " +
-				"allowing executing all native terraform commands inside the shell without using atmos-specific arguments and flags")
-			u.PrintMessage(" - double-dash '--' can be used to signify the end of the options for Atmos and the start of the additional " +
-				"native arguments and flags for the 'terraform' commands. " +
-				"For example: atmos terraform plan <component> -s <stack> -- -refresh=false -lock=false")
```

Comment:
We should document the `--` behavior.

Reply:
Knowing the section where we should add this info given that it is atmos wide concept would actually be beneficial.

Maybe we could do this at the footer of the help.
But let me know if you have any different thoughts

Reply:
When we merge:
- cloudposse/atmos#853

We should do this:
https://github.com/cloudposse/atmos/pull/853/files#diff-d0c4403ba54f3c609af975b7773ae2d4b780f6439bdec1919e70a94b5961ebfd